pkg/vpcmodel/nodesExplainability.go

```diff
@@ -75,6 +75,14 @@ func (e *ExplanationArgs) getConnectionSet() *common.ConnectionSet {
 	return connection
 }
 
+func (e *ExplanationArgs) GetSrc() string {
+	return e.src
+}
+
+func (e *ExplanationArgs) GetDst() string {
+	return e.dst
+}
+
```

Comment:
use golang conventions for getters, see: https://go.dev/doc/effective_go#Getters